src/style.css

```diff
@@ -52,18 +52,18 @@ body {
     background-color: rgb(25, 50, 76);
 }
 
-.scroll-icon{
-     position: absolute;
-     right: 20px;
-     cursor: pointer;
-     z-index: 99;
-     bottom: 85px;
-     background-color: rgb(66, 133, 244);
-     padding: 0px 8px;
-     border-radius: 50%;
-     padding-top: 5px;
-     display: none;
- }
+.scroll-icon-bottom, .scroll-icon-top{
+    position: absolute;
+    right: 20px;
+    cursor: pointer;
+    z-index: 99;
```

Comment:
Just remove this line, and squash your commits. z-index is already being added inline (and set to 1).